README.md

```diff
@@ -523,6 +523,7 @@ If you are using ProGuard you might need to add the following options:
 -keepclasseswithmembers class * {
     @com.squareup.moshi.* <methods>;
 }
+-keep @com.squareup.moshi.JsonQualifier class *
```

Comment:
Yep, seems to work on inner classes of the adapter too. But yeah, I dunno if there is some ProGuard keyword for just interfaces